bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -1396,494 +1309,87 @@ void deserial_free_elem(void *value)
     return;
 }
 
-int get_outer_inner_map_infos(
-    int *inner_fd, struct bpf_map_info *inner_info, int *outter_fd, struct bpf_map_info *outter_info)
+int get_map_infos()
 {
-    int ret;
-    unsigned int outter_id, inner_id;
-
-    outter_id = get_map_id("OUTTER_MAP_ID");
-    inner_id = get_map_id("INNER_MAP_ID");
-    if (!outter_id || !inner_id)
-        return -1;
-
-    ret = get_map_fd_info(inner_id, inner_fd, inner_info);
-    ret |= get_map_fd_info(outter_id, outter_fd, outter_info);
-    if (ret < 0 || map_info_check(outter_info, inner_info))
-        return -1;
-    return 0;
-}
-
-void outter_map_insert(struct task_contex *ctx)
-{
-    int i, end, idx, ret = 0;
-    pthread_t tid = pthread_self();
-
-    i = (ctx->task_id * TASK_SIZE) ? (ctx->task_id * TASK_SIZE) : 1;
-    end = ((i + TASK_SIZE) < MAX_OUTTER_MAP_ENTRIES) ? (i + TASK_SIZE) : MAX_OUTTER_MAP_ENTRIES;
-    for (; i < end; i++) {
-        idx = g_inner_map_mng.elastic_slots[i];
-        if (!g_inner_map_mng.inner_maps[idx].map_fd) {
-            continue;
-        }
-
-        ret = bpf_map_update_elem(ctx->outter_fd, &idx, &g_inner_map_mng.inner_maps[idx].map_fd, BPF_ANY);
+    int i, ret;
+    unsigned int inner_id;
+    char *map_names[] = {
+        "Map64",
+        "Map192",
+        "Map296",
+        "Map1600",
```

Comment:
nit: I am not sure why select 296 and 1600, is there byte alignments then we need tp adjust to 2^N